connexion/api.py

```diff
@@ -164,7 +171,8 @@ def add_paths(self, paths=None):
                         import sys
                         logger.error(error_msg)
                         et, ei, tb = sys.exc_info()
-                        raise ei.with_traceback(tb)
+                        ei.__traceback__ = tb
```

Comment:
That change was made in another PR. So it will cause a merge conflict for this branch after the #162 gets merged or when it get merged depending on what comes first.